### ISSUE TITLE
Add fuseki server as in-jvm mount state

### ISSUE THREAD

**Issue by theferrit32** [clingen-data-model/genegraph#118]
This will enable running the fuseki web server as a dev sparql interface to the database while genegraph holds the database lock. The Clojure namespace this is implemented in can be excluded from production build with leiningen profile.

Fuseki can be run in Java embedded mode and it seems can be limited to read-only access to the dataset.

https://jena.apache.org/documentation/fuseki2/fuseki-embedded.html#fuseki-basic

**Comment by larrybabb** [comment]
@toneillbroad could you please add the final outcome to this and close it ?

**Comment by toneillbroad** [comment]
The crux of this story is to run the Fuseki UI in the same VM as Genegraph as it allows users to issue raw SPARQL queries, provides syntax error highlighting and provides results as raw triples. Ideally, a user could also explore the nuances of the varied syntax of embedded Lucene text searching. Running in the same VM is the only way documented by Apache Jena authors for getting around data store locking issues. 

There is a version of Fuseki that was called “main” that was later termed “server” (I may have that reversed) that I was able to run in the Genegraph VM. However it exposes only endpoints with no UI. I did have to provide a modified version of the genegraph-assembly.ttl config file to the “server” version as there was a locking problem trying to access the Lucene text index. But when I commented out the text section in the modified assembly file I was at least able to issue queries against the raw endpoint and receive responses. 

In pursuit of getting the UI version (called “full”) running in Genegraph I thought I’d try a couple of different approaches: deploy the raw war as part of the embedded pedestal server, run the full server in a separate thread, and finally as a suggestion from Tristan when I was unsuccessful with those, perhaps use an external program Twinkle to access the embedded servers endpoints. All were unsuccessful, and too much time was spent on this so we kicked this to the curb for now.
